justfile

```diff
@@ -1,12 +1,27 @@
-test:
-    docker run \
+test PATH="":
+    docker build -t chm_utils . -f Dockerfile.test && \
+        docker run \
         -it \
-        --rm \
-        -v $HOME/.aws/credentials:/root/.aws/credentials \
-        -e AWS_DEFAULT_REGION=eu-west-1 \
-        -e AWS_PROFILE=chm-admin \
-        -v .:/home/chm_utils/. \
-        -w /home/chm_utils \
+        -v $(pwd)/tests/:/home/tests \
+        -v $(pwd)/chm_utils:/home/chm_utils \
+        -v $HOME/.aws/credentials:/root/.aws/credentials:ro \
+        --env-file .env.test \
```

Comment:
how about we replace this with `--env-file $HOME/.snowflake/.env ` so we can reuse the same credentials across projects?